Sources/MongoSwift/MongoClient.swift

```diff
@@ -357,9 +358,10 @@ public class MongoClient {
      */
     public func listDatabases(
         _ filter: Document? = nil,
+        _ options: ListDatabasesOptions = ListDatabasesOptions(),
```

Comment:
let's use a default value of `nil` here and below, for consistency with the rest of our API.

also, let's require the parameter label here and below. right now the underscore in `_ options` says users don't need to specify it, but in the rest of our API we do require the label.

Reply:
here's some relevant reading on argument labels: https://swift.org/documentation/api-design-guidelines/#argument-labels

we generally stick to this, with the addendum that if there is a first parameter where the meaning is already fairly clear from context (such as the filter here) we drop it.